src/styles/main.scss

```diff
@@ -12,16 +23,20 @@ html {
 body {
     display: flex;
     flex-direction: column;
-    font-size: 1.4rem;
+    font-size: 1.8rem;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
```

Comment:
I did learn something new today. You are right, `subpixel-antialiased` works better on light background with dark font colors. However we would need to add `antialiased` to most areas on homepage i.e: banners 3rd quote (orange section), buckets, footer. 

The font does not render correctly on both retina or non-retina. Here is a demo.

<img width="656" alt="screenshot 2016-02-15 15 33 36" src="https://cloud.githubusercontent.com/assets/12257565/13060788/88ac2028-d3f9-11e5-98e5-821449c4a50c.png">

We can have something like this 

`.font-smoothing {
	-webkit-font-smoothing: antialiased;
	-moz-osx-font-smoothing: grayscale;
}`

and add that to areas with white text over colored backgrounds.